### ISSUE TITLE
Divider should be hidden from assitive technologies

### ISSUE THREAD

**Issue by dougmacknz** [cultureamp/kaizen-design-system#1920]
TLDR: the `<hr>` in the `Divider` component just needs `aria-hidden="true"`

--------

## Audit info

Type:WCAG 2
Principle: Perceivable
Guideline:1.1 Text Alternatives
Success Criteria: 1.1.1 Non-text Content (A)
Level: A
Platform: Web
Severity:Low
Element: Horizontal rule
Component: Main content
Page: Survey Summary
URL: https://intopia.cultureamp.com/app/home
Intopia Key: CUL-147

### What's the issue?
The horizontal rule used on the page is not marked up as a decorative element.
Note: this relates to all `<hr>` elements within the page

### Why it matters
A screen reader will announce the separator if it is not marked as decorative. This adds an unnecessary step to the user’s navigation path.

### Our recommendation
Add aria-hidden="true" to the `<hr>` element to hide it from screen readers.

**Comment by sentience** [comment]
Makes me wonder what value we're getting out of the `hr` element in the first place. Is a styled `div` better than an `hr aria-hidden=true`?

**Comment by dougmacknz** [comment]
> Is a styled div better than an hr aria-hidden=true

As far as accessibility goes, I believe those two things would have the same result. I know `<hr>` has some cross browser issues, but it doesn't seem like we've had any.

Given we've already done the work to style the `<hr>`, I'd say let's just quickly add the `aria-hidden` and be done, but I'd also be fine with refactoring this if someone felt like doing that for some reason as well.
